src/input.rs

```diff
@@ -101,6 +108,13 @@ impl Input {
         });
         self.0.mouse_pos = pos;
     }
+
+    pub fn mouse_wheel(&mut self, delta: MouseScrollDelta) {
+        self.1.mouse_wheel += match delta {
```

Comment:
The reason I don't accumulate moues offsets in the same way is because the action user might want to do based on the moves is not necessarily cumulative. For example, consider that the user wants to play a sound on each scroll over a threshold. If you just sum up the events, and they end up being near zero, the user will not register the threshold.

TL;DR: I think we should keep a list of the wheel scrolls, and have a method that returns the total